example/src/components/formExample.js

```diff
@@ -18,7 +19,7 @@ export default React.createClass({
 
     getInitialState() {
         return {
-            showCode: false,
+            showCode: this.props.showCode || false,
```

Comment:
Curious as to why would you want to pass this in? 

Also [Props in getInitialState Is an Anti-Pattern](https://facebook.github.io/react/tips/props-in-getInitialState-as-anti-pattern.html)